modules/persistent-storage-csi-olm-operator-uninstall.adoc

```diff
@@ -8,7 +8,7 @@
 [id="persistent-storage-csi-olm-operator-uninstall_{context}"]
 = Uninstalling the {FeatureName} CSI Driver Operator
 
-All EFS PVs are inaccessible after uninstalling the AWS EFS CSI Driver Operator.
+All EFS PVs are inaccessible after uninstalling the Amazon EFS CSI Driver Operator.
```

Comment:
Revoked.

Reply:
@smunje1 It doesn't seem like this title is linking out to OCP resources or code. "Amazon EFS CSI driver" is the appropriate name to be used for the driver. Is there a reason I'm missing why this update should be revoked?

https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html

Reply:
Updated, "All EFS PVs are inaccessible after uninstalling the [AWS EFS CSI Driver Operator](https://github.com/openshift/aws-efs-csi-driver-operator) (a Red Hat operator)."